src/test/java/test/dependent/DependentTest.java

```diff
@@ -152,38 +154,61 @@ public void methodDependencyBetweenClassesShouldWork(Class[] classes, boolean pr
   @DataProvider
   public static Object[][] dp1380() {
     return new Object[][] {
-      {GitHub1380Sample.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}, false},
-      {GitHub1380Sample2.class, new String[] {"testMethodC", "testMethodB", "testMethodA"}, false},
-      {GitHub1380Sample3.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}, false},
-      {GitHub1380Sample4.class, new String[] {"testMethodB", "testMethodA", "testMethodC"}, false},
-      {GitHub1380Sample.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}, true},
-      {GitHub1380Sample2.class, new String[] {"testMethodC", "testMethodB", "testMethodA"}, true},
-      {GitHub1380Sample3.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}, true},
-      {GitHub1380Sample4.class, new String[] {"testMethodB", "testMethodC", "testMethodA"}, true}
+      {GitHub1380Sample.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}},
+      {GitHub1380Sample2.class, new String[] {"testMethodC", "testMethodB", "testMethodA"}},
+      {GitHub1380Sample3.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}},
+      {GitHub1380Sample4.class, new String[] {"testMethodB", "testMethodA", "testMethodC"}},
     };
   }
 
   @Test(dataProvider = "dp1380", description = "GITHUB-1380")
-  public void simpleCyclingDependencyShouldWork(
-      Class<?> testClass, String[] runMethods, boolean isParallel) {
+  public void simpleCyclingDependencyShouldWorkWithoutParallelism(
+      Class<?> testClass, String[] runMethods) {
```

Comment:
`String... runMethods` will simplify the data provider.

Reply:
I think its going to be the same thing because its just another way of representing an array